.changeset/wild-eyes-roll.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-app-api': minor
```

Comment:
Should be a `patch` since this is not a breaking change and we're on 0.x

Reply:
👍 changed to `patch`